spec/models/post_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+# Checks if all errors come from given attribute , returns false if otherwise
+def TestAttributeOfPost(obj , attribute)
+  # obj : object to check the attributes from
+  # attribute: attribute name of the given object to check errors on
+  obj.valid?
+  error_count = obj.errors.count
+  attribute_err_count = obj.errors[attribute].count
+  if(attribute_err_count < error_count)
+    return false
+  else
+    return true
+  end
+end
+
+RSpec.describe Post, type: :model do
+  let(:post) {FactoryBot.build :post}
+
+  describe "is valid" do
+
+    it "with title and description present" do
+      expect(post).to be_valid
+    end
+
+  end
+
+  describe "is not valid" do
+
+    it "with title missing" do
+      post.title = nil
```

Comment:
šeit tad vajadzetu būt:
post.title = nil
expect(post).to_not be_valid

Un jā nestrada, tad
expect(post.valid?).to eq(false)

Reply:
Kāpēc jāizmanto šo test_attribute_of_post ?
Kāpēc expect(post).to_not be_valid nestrādā?

Reply:
strada , tikai ta metode(test_attribute_of_post)  parbauda tiesi vai tas viens attributs atbild par visiem errors un vai nav kads cits pie vainas kam nevaidzetu.